python/ray/train/_internal/backend_executor.py

```diff
@@ -111,6 +111,12 @@ def start(
             actor_cls_kwargs=train_cls_kwargs,
             placement_group=placement_group,
         )
+        if self._trial_info:
```

Comment:
Do we still need this if statement?

Reply:
yes, added a comment here to clarify.

# RLlib's LearnerGroup also uses BackendExecutor but without TrialInfo.